### ISSUE TITLE
Crash on Android after scanning when using TabView

### ISSUE THREAD

**Issue by tigrenok00** [EddyVerbruggen/nativescript-barcodescanner#73]
Hi,
Just after the scan the barcode is recognized and displayed in the bottom of the screen, but before the alert message is able to be displayed (I'm using the code in the example on git) - the app crushes with:
"Unable to resume activity", see screenshot:

![screenshot_20170202-171417](https://cloud.githubusercontent.com/assets/23075641/22555511/9d170daa-e96c-11e6-8740-025e4ff6d021.png)

I'm using HTC M8

**Comment by EddyVerbruggen** [comment]
Which example do you refer to? I see `org.nativescript.groceries` in your log.. which is not the ID of the demo in this repo.

Also, what happens when you change the `alert` to `console.log`?

**Comment by tigrenok00** [comment]
Thanks for the reply!
I'm using this tutorial here as a basis https://docs.nativescript.org/tutorial/ng-chapter-0, and the piece of code for example of usage from your repo. 
I will let you know about console.log after I check.

**Comment by tigrenok00** [comment]
Hi, it is the same with console.log.
Is there any data I can provide for you to try to figure this out?
Thank you

**Comment by tigrenok00** [comment]
By the way, if I hit 'back' after the scanner is open but before pointing to a barcode to scan, it gives the same error

**Comment by EddyVerbruggen** [comment]
Can you please share your repo soI can clone and reproduce the issue?

**Comment by tigrenok00** [comment]
https://github.com/tigrenok00/groceries-2
Please try this repo, you should be able to reproduce it here.
Thanks!

**Comment by EddyVerbruggen** [comment]
Looks like there's bits missing in your repo:

<img width="1006" alt="screen shot 2017-02-04 at 12 48 40" src="https://cloud.githubusercontent.com/assets/1426370/22618269/5b61fbaa-ead8-11e6-89f2-e2ad815ca9d4.png">

**Comment by tigrenok00** [comment]
Sorry, I removed the unnessecary components and those were left though not referenced. It should be okay now.

**Comment by EddyVerbruggen** [comment]
Thanks, that's better!

**Comment by EddyVerbruggen** [comment]
As the stacktrace in the screenshot indicates there's a problem with the `TabView` component. Replace the contents of `orders.html` by this avoids the issue. Trying to find the cause..

```html
<StackLayout>
  <Button text="Scan!" (tap)="scan()"></Button>
</StackLayout>
```

**Comment by EddyVerbruggen** [comment]
Hi @tigrenok00, it looks like you're doing everything right and the plugin is not to blame either. I found a closed bug on the NativeScript repo where someone had a similar issue but it was never fixed. I reopened it with a proposed fix. In the meanwhile try not to use the barcode scanner in combination with Angular, a TabView, and LiveSync enabled.

See https://github.com/NativeScript/NativeScript/issues/3201#issuecomment-277466385 for details.
